### DISCUSSION TITLE
桌面图标缩小和相机返回动画丢失

### DISCUSSION THREAD

**Discussion by Sev73n** [MonwF/customiuizer#519] [General]
小米14 正式版1.0.45
应用用侧滑返回退出到桌面 桌面图标会缩小
需要上滑手势才能正常
相机应用返回手势回到桌面 桌面图标变小且返回动画丢失
测试了关闭米客lopsed就没问题了
应该是米客的bug

**Comment by MonwF** [comment]
关闭桌面作用域再试试。。自己先排查下哪个功能吧
